packages/at_chops/lib/at_chops.dart

```diff
@@ -1,34 +1,16 @@
-library at_chops;
+// export cryptography wrappers
+export 'src/hashing/hashing.dart';
```

Comment:
Added files named after the directories in src to make export management easier.